mol-loader/rdkit_utils.py

```diff
@@ -1,79 +1,89 @@
-from rdkit import Chem

```

Comment:
Argh. I don't know what's going on with whitespace in this file. I'll fix it on master.